hack/common.sh

```diff
@@ -11,8 +11,14 @@ set -o pipefail
 readonly OS_ROOT=$(
   unset CDPATH
   os_root=$(dirname "${BASH_SOURCE}")/..
+  
   cd "${os_root}"
-  pwd
+  os_root=`pwd`
+  if [ -h "${os_root}" ]; then
+    readlink "${os_root}"
```

Comment:
They do but it's not the same readlink.  Since you're checking for a symbolic, this is probably ok.